### ISSUE TITLE
Day name is too big to show on home screen

### ISSUE THREAD

**Issue by vitor9** [amirzaidi/Launcher3#517]
This problem has bothering me for a while, I am from Brazil and we have day names that are longer than usual. Monday, for example is called "segunda-feira", it has 13 characters and can barely show together with the number of the day and weather.
A soltion I can give is, just make it shorter for 3 characters, each day would be like:
- Monday is Segunda-feira, short should be Seg
- Tuesday is Terça-feira, short should be Ter
- Wednesday is Quarta-feira. short should be Quar
- Thursday is Quinta-feira, short would be Qui
- Friday is Sexta-feira, short would be Sex
- Saturday is Sabado, it isn't that long but since the others are already shorter, Sab.
- Sunday is Domingo, same rule for Saturday, Dom.

**Comment by coruja747** [comment]
indeed, I confirm this. Here on my phone (galaxy S7 flat) the pixel launcher rarely show date text correctly. Most dates get cropped, like today, where `Quarta-feira, 18 de julho` turned into `Quarta-feira, 18 de j...`

I found [this pic](https://plus.google.com/photos/photo/109607167926686908721/6559524114645530498?sqid=106833864784947425671&ssid=b45f6230-403e-4c53-9d3e-fc71b944ff31) of an Pixel 2 phone running android P in pt-br language, which confirm that **google official solution** for this is just abbreviate the date text (to be more specific, only the month name should be abbreviated, eg: `Quarta-feira, 18 de julho` should be `Quarta-feira, 18 de jul`)
